src/main/java/Duke.java

```diff
@@ -1,10 +1,161 @@
+            System.out.println("There are no tasks!");
+            System.out.println(HORIZONTAL_LINE);
+            return;
+        }
+
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Here's the task list: ");
+
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println((i + 1) + ". "  + tasks[i]);
+        }
+
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void markTask(Task[] tasks, int taskIndex, boolean isDone) {
+        System.out.println(HORIZONTAL_LINE);
+        Task taskToEdit = tasks[taskIndex];
+
+        if (taskToEdit.isDone == isDone) {
+            System.out.println("Task is already marked as " + (isDone ? "done!" : "undone!"));
+            System.out.println(HORIZONTAL_LINE);
+            return;
+        }
+
+        else if (isDone) {
+            System.out.println("Nice! I've marked this task as done: ");
+            taskToEdit.setStatus(true);
+        }
+
+        else {
+            System.out.println("OK, I've marked this task as not done yet: ");
+            taskToEdit.setStatus(false);
+        }
+
+        System.out.println(taskToEdit);
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static String removeFirstWord(String input){
+        return input.split(" ",2)[1];
+    }
+
+    public static void printAcknowledgement(String taskType, String description, int taskCount){
+        System.out.println(HORIZONTAL_LINE);
+        System.out.println("Added " + taskType + ": " + description);
+        System.out.println("Total Number of Tasks: " + taskCount);
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static String getDeadlineDescription(String input) {
+        return input.split(" /by ",2)[0];
+    }
+
+    public static String getDeadlineDate(String input) {
+        return input.split(" /by ", 2)[1];
+    }
+
+    public static String getEventDescription(String input) {
+        return input.split(" /from ",2)[0];
+    }
+
+    public static String[] getEventTimings(String input) {
+
+        String[] timings = new String[2];
+        String timing = input.split(" /from ", 2)[1];
+
+        timings[0] = timing.split(" /to ", 2)[0];
+        timings[1] = timing.split(" /to ", 2)[1];
+
+        return timings;
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        startUp();
+
+        Scanner in = new Scanner(System.in);
+        String inputLine = in.nextLine();
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        while (true) {
+            String[] words = inputLine.split(" ");
+
+            if (words[0].equals("bye")) {
```

Comment:
Perhaps you can consider using .startsWith? So that you can directly do "inputLine.startsWith("bye")" for example.